packages/orbit-components/src/Heading/index.tsx

```diff
@@ -44,9 +46,10 @@ const Heading = ({
 
   return (
     <Component
+      aria-level={Component === "div" ? level : undefined}
       id={id}
       data-test={dataTest}
-      role={Component === "div" ? "heading" : undefined}
+      role={Component === "div" ? role : undefined}
```

Comment:
I would add these new props (`aria-level`, `role`) to Storybook as well (Playground story).

Reply:
There are conditional controls in SB, however, it's limited and we can't include two conditions for one control (show `level` when `as="div"` **and** `role="heading"`). :(

Reply:
Precisely, I'd avoid unnecessary complication for something users can't see 😕